extension/webviews/components/ViewerTopBar.svelte

```diff
@@ -12,9 +12,9 @@
     $page = Page.InviteUser;
   };
 
-  const showInviteButton = $page === Page.ThreadsViewer || $page === Page.ReplyViewer;
+  $: showInviteButton = $page === Page.ThreadsViewer || $page === Page.ReplyViewer;
 
-  const dropdownOptions = [
+  $: contextMenuDropdownOptions = [
```

Comment:
had to make this reactive as well since it's dependent on a reactive var. [svelte docs](https://svelte.dev/docs/svelte-components#script-3-$-marks-a-statement-as-reactive)